app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -57,7 +57,6 @@ import org.simple.clinic.rules.LocalAuthenticationRule
 import org.simple.clinic.storage.text.TextStore
```

Comment:
There's an error in the commit message. "there" is being used when we should be using "their". Please correct it?